### ISSUE TITLE
"Pop out stage" feature

### ISSUE THREAD

**Issue by towerofnix** [LLK/scratch-gui#2225]
Just a quick suggestion idea - I saw [this post](https://scratch.mit.edu/discuss/topic/302843/) on the discussion forums:

>Hi,
>
>I use multiple screens for my computer. Is it possible to modify the Scratch work-space? Such as having the stage, and blocks on separate screens?
>
>Thanks!

This would be possible if the stage could be popped out into a separate browser window (which is certainly doable with JS). I imagine the sprite palette would expand to fill up the space of the stage.

**Comment by towerofnix** [comment]
@kyleplo Yup, just like that. And I was imagining the sprite list would go exactly as in your screenshot; that is, just taking up some space on the right (maybe a bit less horizontal space, since there's more vertical to make up for it). But obviously this is just a rough suggestion, so if anyone has their own idea I'd be glad to hear it :)

**Comment by GreenScripter** [comment]
It would be really neat if you could drag out and rearrange all the different parts of the view, kind of like some normal IDEs, but it might be a little impractical or confusing in scratch.
